doc/kernel/other/ring_buffers.rst

```diff
@@ -303,22 +303,10 @@ operations on the ring buffer's memory.  For example:
 	...
     }
 
-APIs
-****
-
-The following ring buffer APIs are provided by :file:`include/ring_buffer.h`:
-
-* :cpp:func:`RING_BUF_ITEM_DECLARE_POW2()`
-* :cpp:func:`RING_BUF_ITEM_DECLARE_SIZE()`
-* :cpp:func:`RING_BUF_DECLARE_SIZE()`
-* :cpp:func:`ring_buf_init()`
-* :cpp:func:`ring_buf_is_empty()`
-* :cpp:func:`ring_buf_space_get()`
-* :cpp:func:`ring_buf_item_put()`
-* :cpp:func:`ring_buf_item_get()`
-* :cpp:func:`ring_buf_put()`
-* :cpp:func:`ring_buf_put_claim()`
-* :cpp:func:`ring_buf_put_finish()`
-* :cpp:func:`ring_buf_get()`
-* :cpp:func:`ring_buf_get_claim()`
-* :cpp:func:`ring_buf_get_finish()`
+API Reference
+*************
+
+The following ring buffer APIs are provided by :zephyr_file:`include/ring_buffer.h`:
```

Comment:
This is rather cool!  I can do a general scan/replace after this PR gets merged for most of the :file:`xxx` references in other documents.  (I have a script I run occasionally to check that :file:`xxx` references are to files that exist.)